packages/core/src/styles/theme.dark.scss

```diff
@@ -0,0 +1,78 @@
+  --cds-alias-object-interaction-background-highlight: #{$cds-global-color-blue-400};
+
+  --cds-alias-status-info: #{$cds-global-color-blue-400};
+  --cds-alias-status-info-tint: #{$cds-global-color-blue-300};
+  --cds-alias-status-info-shade: #{$cds-global-color-blue-500};
+  --cds-alias-status-success: #{$cds-global-color-green-500};
+  --cds-alias-status-success-tint: #{$cds-global-color-green-300};
+  --cds-alias-status-success-shade: #{$cds-global-color-green-600};
+  --cds-alias-status-warning: #{$cds-global-color-ochre-400};
+  --cds-alias-status-warning-tint: #{$cds-global-color-ochre-300};
+  --cds-alias-status-warning-shade: #{$cds-global-color-ochre-500};
+  --cds-alias-status-danger: #{$cds-global-color-red-500};
+  --cds-alias-status-danger-tint: #{$cds-global-color-red-300};
+  --cds-alias-status-danger-shade: #{$cds-global-color-red-600};
+  --cds-alias-status-neutral: #{$cds-global-color-construction-400};
+  --cds-alias-status-neutral-tint: #{$cds-global-color-construction-300};
+  --cds-alias-status-neutral-shade: #{$cds-global-color-construction-500};
+  --cds-alias-status-alt: #{$cds-global-color-violet-300};
+  --cds-alias-status-alt-tint: #{$cds-global-color-violet-200};
+  --cds-alias-status-alt-shade: #{$cds-global-color-violet-400};
+  --cds-alias-status-disabled: #{$cds-global-color-construction-600};
+  --cds-alias-status-disabled-tint: #{$cds-global-color-construction-500};
+  --cds-alias-status-disabled-shade: #{$cds-global-color-construction-700};
+  --cds-alias-status-secondary-tint: transparent;
+  --cds-alias-status-secondary-info: #{$cds-global-color-blue-800};
+  --cds-alias-status-secondary-info-tint: #{$cds-global-color-blue-700};
+  --cds-alias-status-secondary-info-shade: #{$cds-global-color-blue-900};
+  --cds-alias-status-secondary-success: #{$cds-global-color-green-800};
+  --cds-alias-status-secondary-success-tint: #{$cds-global-color-green-700};
+  --cds-alias-status-secondary-success-shade: #{$cds-global-color-green-900};
+  --cds-alias-status-secondary-warning: #{$cds-global-color-ochre-800};
+  --cds-alias-status-secondary-warning-tint: #{$cds-global-color-ochre-700};
+  --cds-alias-status-secondary-warning-shade: #{$cds-global-color-ochre-1000};
+  --cds-alias-status-secondary-danger: #{$cds-global-color-red-800};
+  --cds-alias-status-secondary-danger-tint: #{$cds-global-color-red-700};
+  --cds-alias-status-secondary-danger-shade: #{$cds-global-color-red-1000};
+  --cds-alias-status-secondary-neutral: #{$cds-global-color-construction-600};
+  --cds-alias-status-secondary-neutral-tint: #{$cds-global-color-construction-500};
+  --cds-alias-status-secondary-neutral-shade: #{$cds-global-color-construction-800};
+  --cds-alias-status-secondary-alt: #{$cds-global-color-violet-800};
+  --cds-alias-status-secondary-alt-tint: #{$cds-global-color-violet-700};
+  --cds-alias-status-secondary-alt-shade: #{$cds-global-color-violet-1000};
+}
```

Comment:
Since we have been able to abstract most of the design system tokens into alias tokens or abstract values these are all the values needed to create the dark theme.